manager/gui/src/router/index.ts

```diff
@@ -57,6 +57,12 @@ const routes: Array<RouteConfig> = [
     component: LrAlert,
     props: true,
   },
+  {
```

Comment:
this is not required

Reply:
Removed the extra route for Alerts page.